packages/graphql-codegen-cli/src/utils/watcher.ts

```diff
@@ -88,15 +88,19 @@ export const createWatcher = (
         }
       });
 
+    let pollingInterval = 100;
+    if (typeof config.watchPolling === 'object') {
```

Comment:
Maybe we can have a function to parse the `config.watchConfig` object and apply defaults?

Reply:
Since the config structure's now identical to chokidar's and I don't think there's any reason to maintain different default values for them, I just inlined the values where we create the watcher